stats/parsing-stats/lang/scala/projects.txt

```diff
@@ -8,7 +8,7 @@ https://github.com/akka/akka
 https://github.com/yahoo/CMAK
 https://github.com/ornicar/lila
 https://github.com/gitbucket/gitbucket
-https://github.com/twitter/finagle
+# https://github.com/twitter/finagle
```

Comment:
would be good to put that back.
Otherwise the 1% improvement from 95.5 to 96.5 could just come from that?